lib/mix/tasks/release_version.ex

```diff
@@ -82,10 +82,11 @@ defmodule Mix.Tasks.Release.Version do
   end
 
   # Sets the release version to the new value by using the passed update funs.
+  # Mix.ProjectStack is GenServer after Elixir 1.10.0-rc.0
```

Comment:
I think this comment is redundant, because from this point onwards you'll find `Mix.ProjectStack` part of `GenServer` anyways. The changes represented in this PR and commit should be enough to quantify that syntax has changed.

Reply:
ok, makes sence